frame/staking/src/lib.rs

```diff
@@ -660,6 +659,11 @@ pub trait Trait: frame_system::Trait {
 	/// For each validator only the `$MaxNominatorRewardedPerValidator` biggest stakers can claim
 	/// their reward. This used to limit the i/o cost for the nominator payout.
 	type MaxNominatorRewardedPerValidator: Get<u32>;
+
+	/// Deprecated Time.
+	///
+	/// Deprecated associated type wrongly used to query Time in millisecond.
+	type DeprecatedTime: Time;
```

Comment:
Why do we need this?

Reply:
the main use it to be able to decode the old structure `ActiveInfo<MomentOf<T>>`.
But indeed I can just decode the EraIndex and not start field as I don't use it.

I updated code as such, (also fixing that StorageVersion wasn't set)